package.json

```diff
@@ -40,7 +40,7 @@
     "!lib/testUtils"
   ],
   "engines": {
-    "node": ">=8.7.0"
+    "node": ">=10"
```

Comment:
maybe better start to support with LTS, i.e. 10.13.0

Reply:
Some users had problem, when we set minor version too high: https://github.com/stylelint/stylelint/issues/4000. But it's not longer the case:

> Starting with Node 10, AWS Lambda will also automatically update the language minor versions to latest minor version, as specified by https://github.com/nodejs/Release. 

https://aws.amazon.com/about-aws/whats-new/2019/05/aws_lambda_adds_support_for_node_js_v10/

So I guess we can even put latest 10.18.0 as the minimum.

Reply:
I think allowing for edge cases like what happened in #4000 is fine, min v10.0.0 I think will be ok

Reply:
Unless  a specific feature is required from a version newer than 10.0.0, we can just go with 10.0.0. Just note that CI tests the latest release line and this can be problematic in some cases. It might be wiser to test the minimum supported Node.js version at the time instead.